docs/synapse/userguides/storm_adv_control.rstorm

```diff
@@ -7,22 +7,31 @@
 Storm Reference - Advanced - Control Flow
 =========================================
 
-Storm includes a number of common programming control flow structures to facilitate more advanced or complex Storm queries. These include:
+Storm includes a number of common programming control flow structures to facilitate more advanced Storm
+queries. These include:
 
+- `Init block`_
+- `Fini block`_
```

Comment:
The lowercase 'b' in block vs uppercase 'S'/'L' in Statement/Loop feels inconsistent to me